### PR TITLE
updated Readme

### PR DESCRIPTION
**Pull request by adeelehsan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,161 +1,135 @@
...
-## Exception Handling
+### Examples
+Complete examples can be found in the [Getting Started notebooks](./examples/01_getting_started).
+
+### Usage
+
+* Querying the corpora
+    ```python
+    from vectara import SearchCorporaParameters, Vectara
+    client = Vectara(
```
Comment (ofermend) [inline README.md:30]: Perhaps explain here that you should use either the API Key or the OAUth ID/secret, but not both at the same time? on first read, that may be confusing to users.

File: README.md [hunk 2]
```diff
@@ -1,161 +1,135 @@
...
+   chat_params = ChatParameters(store=True)
+
+   session = client.create_chat_session(
+       search=search_params,
+       generation=generation_params,
+       chat_config=chat_params,
+   )
+   
+   response = session.chat(query="Tell me about machine learning.")
+   print(response.answer)
```
Comment (ofermend) [inline README.md:73]: Here please show how someone does a subsequent chat. i.e. using the returned Chat session for a new turn

**Comment by adeelehsan** [comment]
@ofermend please have a look again.

File: README.md [hunk 3]
```diff
@@ -1,161 +1,163 @@
...
+Complete examples can be found in the [Getting Started notebooks](./examples/01_getting_started).
+
+### Usage
+
+* Querying the corpora <br />
+  You can use either `api_key` or `client_id` and `client_secret` for authentication.
+    ```python
+    from vectara import SearchCorporaParameters, Vectara
+    client = Vectara(
+        api_key="YOUR_API_KEY", # Use this for API key-based authentication
```
Comment (ofermend) [inline README.md:32]: This is better but not ideal still. My recommendation is:
1. Have a section about "create a client" that just shows the client=Vectara() piece, and shows it in two different ways (not a single call with "OR" comment - that can be confusing).
2. Then show query (i.e. setup search params, generation params, and the query). But please add some parameters there as an example. IT needs to be a good example where someone can copy/paste and get good results. You can then suggest they can look at docs for more details and options.
3. Chat example.

File: README.md [hunk 4]
```diff
@@ -1,161 +1,163 @@
...
+    
+   search = SearchCorporaParameters(
+       corpora=[...],
+       ...
+   )
+   generation = GenerationParameters(...)
+   chat_params = ChatParameters(store=True)
+
+   session = client.create_chat_session(
+       search=search_params,
```
Comment (ofermend) [inline README.md:69]: shouldn't this be search=search and generation=generation? (search_params is not defined).
Please make these real examples and test them to validate.

**Review by ofermend — APPROVED** [review]
LGTM. 
There are a few minor things in formattting, but let's merge and I can do a small PR to fix later.

**Pull request merged**